ZhuoHeiChaAPI/Services/GameService.cs

```diff
@@ -11,13 +12,57 @@ namespace ZhuoHeiChaAPI.Services
 {
     public class GameService : IGameService
     {
+        private Dictionary<int, Dictionary<int, List<PayerInfo>>> _returnTable = new Dictionary<int, Dictionary<int, List<PayerInfo>>>();
```

Comment:
Consider adding a line of comment above stating what the key-value structure is for better readability. For example, 
`<gameId, <receiverId, PayerInfo>>`

Reply:
Done